lib/samr21_xpro/sensor.c

```diff
@@ -0,0 +1,89 @@
+#include <stdio.h>
```

Comment:
(I cannot comment right there, but) you added a .swp file :)

But don't worry I got you something ;)
```
set directory=~/.cache/vimswp   " Don't clutter working dirs with swap files
if !isdirectory($HOME . '/.cache/vimswp')
    call mkdir($HOME . '/.cache/vimswp')
endif
```